knimin/lib/tests/test_mail.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase, main
+
+from knimin.lib.mail import send_email
+
+
+class TestEmail(TestCase):
+    def test_send_email_basic(self):
```

Comment:
Does this means that the system will send an email to `'americangut@gmail.com'` every time that the tests are executed?